Backend/server.js

```
@@ -10,14 +9,16 @@ const retailerOrderRoutes = require("./routes/retailerOrderRoutes");
 const customerOrderRoutes = require("./routes/customerOrderRoutes");
 const shopRoutes = require("./routes/shops");
 const productRoutes = require("./routes/productRoutes");
+const cartRoutes = require("./routes/cartRoutes"); // Added cart routes
 const path = require("path");
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)